main.py

```diff
@@ -294,6 +293,10 @@ def start(self, settings):
 				self._qmp = QEMUMonitorProtocol(('localhost', 4444))
 				self._qmp.connect()
 			except Exception as e:
+				# Detect whether XQEMU quit and abort
```

Comment:
Is this supposed to be "quit *or* abort", or am I misinterpreting what this comment means?

Reply:
It was meant like "if xqemu quits, we abort (the loop)". I modified the comment to be a bit more expressive.

Reply:
Ironically I thought the comment in your GitHub response was a lot more readable than either of the code comments.

The new comment is good-enough though. Consider it a nit (I prefer the comment you made here).

Reply:
Ich changed it to something similar to my GitHub response.